src/main/java/werkIt/WerkIt.java

```diff
@@ -69,6 +69,13 @@ public void startContinuousUserPrompt() {
                 } catch (InvalidCommandException e) {
                     System.out.println(e.getMessage());
                     System.out.println("Please try again.");
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    // Comment to be removed: This will catch commands with empty descriptors. Example "workout /new".
+                    // This is due to the "arguments = null" in createWorkoutCommand() method under Parser.java.
+                    // The return new WorkoutCommand at that method will treat arguments as null and will
+                    // NOT throw an InvalidCommandException because a null argument is still valid.
```

Comment:
I'm not sure if this comment is meant to be removed before the PR or not haha, but just highlighting this in case.

Reply:
Yep, it's meant to be removed! Just placed there for justification on why I did the following code. Thanks!